docs/state.md

```diff
@@ -9,12 +9,12 @@ While Linux is usable on all T2 models, some features are limited due to the lac
 - USB
 - Keyboard
 - Camera
-- Wi-Fi
+- Wi-Fi (requires macOS firmware)
+- Bluetooth (requires macOS firmware only for devices with BCM4377 chip)
 - Touch Bar: There is support for the "Touch Bar Keyboard" device configuration, where only the Function Keys or the Media/Brightness Control Keys can be shown. No other graphics can be shown on the Touchbar (this is what Windows with Bootcamp drivers uses).
 
 ## Partially Working
 
-- Bluetooth: Not working on models with the BRCM4377 WiFi chip (MacBookPro15,4, MacBookPro16,3, MacBookAir9,1).
```

Comment:
Maybe mention the issue where it doesn't work well on these models when 2.4GHz Wi-Fi is used at the same time

Reply:
Was it 2.4Ghz or 5HGhz?

Reply:
2.4:

> There is, however, one caveat: WiFi/Bluetooth coexistence isn’t properly configured yet, so you will have poor Bluetooth performance if you are connected to a 2.4GHz WiFi network. We recommend turning off WiFi or using a 5GHz network if you want to use Bluetooth at this time; we will be adding coexistence support in the coming weeks.

https://asahilinux.org/2022/07/july-2022-release/